backend/src/main/kotlin/blue/mild/covid/vaxx/error/ExceptionHandling.kt

```diff
@@ -50,26 +56,21 @@ fun Application.installExceptionHandling() {
             call.errorResponse(HttpStatusCode.NotFound, cause.message)
         }
 
-        exception<ValidationException> { cause ->
-            logger.debug { cause.message }
-            call.errorResponse(HttpStatusCode.BadRequest, "Bad request: ${cause.message}.")
-        }
+        deserializationExceptions()
```

Comment:
to `deserializationExceptions` neni moc anglicky 🧐

Reply:
no `deserialization` je slovo a `exceptions` taky 😄  Z muze byt https://cheatsheetseries.owasp.org/cheatsheets/Deserialization_Cheat_Sheet.html

nicmene prejmenuju